config/routes.rb

```diff
@@ -1,8 +1,9 @@
 Rails.application.routes.draw do
-  root  'urls#new'
+  root  'urls#index'
 
-  resources :urls, only: [:new, :create, :update, :destroy, :show]
-  get '/urls', to: "urls#new"
+  resources :urls, only: [:index, :create, :update, :destroy, :show]
+  get '/urls/:id/:user_token', to: "urls#show"
+  get '/urls', to: "urls#index"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.